core/src/main/java/com/rudderstack/android/sdk/core/RudderMessage.java

```diff
@@ -199,4 +202,12 @@ void updateContext() {
     public String getGroupId() {
         return this.groupId;
     }
+
+    /**
+     * @return Integrations Map passed for the event
+     */
+    @NonNull
+    public Map<String,Boolean> getIntegrations() {
+        return this.integrations;
```

Comment:
we can make the integrations string,object to make it generic

Reply:
Great idea, implemented it